test/tee_pipeline_test.exs

```diff
@@ -0,0 +1,50 @@
+defmodule TestPipeline do
+  @moduledoc false
+  use ExUnit.Case, async: true
+  use Bunch
+
+  alias Membrane.Buffer
+  alias Membrane.Event.EndOfStream
+  alias Membrane.Testing.Pipeline
+  alias Membrane.Testing.Sink
+  alias Membrane.Testing.Source
+
+  def make_pipeline(generator) do
+    Pipeline.start_link(%Pipeline.Options{
+      elements: [
+        src: %Source{actions_generator: generator},
+        tee: Membrane.Element.Tee.Filter,
+        sink1: %Sink{target: self()},
+        sink2: %Sink{target: self()}
+      ],
+      monitored_callbacks: [],
+      test_process: self(),
+      links: %{
+        {:src, :output} => {:tee, :input},
+        {:tee, :output_static} => {:sink1, :input},
+        {:tee, :output_dynamic} => {:sink2, :input}
+      }
+    })
+  end
+
+  def generator_from_data(data) do
+    actions = Enum.map(data, fn element -> {:buffer, {:output, %Buffer{payload: element}}} end)
+
+    fn cnt, size ->
+      if(cnt != 0, do: cnt, else: actions)
+      |> Enum.split(size)
+      ~>> ({to_send, []} -> {to_send ++ [{:event, {:output, %EndOfStream{}}}], []})
+    end
+  end
+
+  test "split input in two outputs" do
```

Comment:
Not sure about this, but `split` sounds like taking two different parts of data, maybe `forward` would be better. Also `in` seems to be incorrect preposition